.eslintrc.yml

```diff
@@ -0,0 +1,7 @@
+env:
+  browser: true
+  commonjs: true
```

Comment:
why do we need this?

Reply:
These configurations are not needed at all and have been removed.